test/recipes/90-test_store.t

```diff
@@ -44,21 +44,32 @@ my @generated_files =
      "rsa-key-pkcs8-pbes2-sha1.pem", "rsa-key-pkcs8-pbes2-sha1.der",
      "rsa-key-sha1-3des-sha1.p12", "rsa-key-sha1-3des-sha256.p12",
      "rsa-key-aes256-cbc-sha256.p12",
-     "rsa-key-md5-des-sha1.p12",
-     "rsa-key-aes256-cbc-md5-des-sha256.p12",
      "rsa-key-pkcs8-pbes2-sha256.pem", "rsa-key-pkcs8-pbes2-sha256.der",
-     "rsa-key-pkcs8-pbes1-md5-des.pem", "rsa-key-pkcs8-pbes1-md5-des.der",
+    );
+my $use_md5 = 0; # does not work: !disabled("md5");
+push(@generated_files, (
+    "rsa-key-md5-des-sha1.p12",
+    "rsa-key-aes256-cbc-md5-des-sha256.p12",
+    "rsa-key-pkcs8-pbes1-md5-des.pem", "rsa-key-pkcs8-pbes1-md5-des.der"
+     )) if $use_md5;
```

Comment:
At the moment `$use_md5` is always 0...so this is redundant code.

Reply:
I've tentatively changed its initialization to
```
my $use_md5 = !disabled("md5");
```
but see the issue with pbeWithMD5AndDES mentioned above.